src/store/events/model.js

```diff
@@ -0,0 +1,19 @@
+const t = require("tcomb-validation");
+
+/**
+ * TODO Maybe use tcomb form ?
```

Comment:
Har spanat in tcomb form lite. Osäker på om den kommer funka bra om vi vill ha lite speciella formulär. Den verkar skapa upp ett formulär med automatiska labels osv. Kan vi styra det lika mycket då tro?